lib/actions.ex

```diff
@@ -3,15 +3,15 @@ defmodule EctoShorts.Actions do
   Actions for crud in ecto, this can be used by all schemas/queries
   """
 
-  @type query :: Ecto.Query | Ecto.Schema
-  @type filter_params :: Keyword.t | map
+  @type queryable :: module | Ecto.Query.t()
+  @type id :: pos_integer()
+  @type filter_params :: Keyword.t() | map
   @type aggregate_options :: :avg | :count | :max | :min | :sum
-  @type schema_list :: list(Ecto.Schema.t) | []
-  @type schema_res :: {:ok, Ecto.Schema.t} | {:error, String.t}
+  @type schema_res :: {:ok, Ecto.Schema.t()} | {:error, String.t()}
```

Comment:
Sorry, my formatter is going buck wild on these brackets. Let me know if this is an issue and I will get it reverted.